tests/integration/test.thrift

```diff
@@ -4,4 +4,5 @@ exception ExpectedException {
 
 service TestService {
     bool example() throws (1: ExpectedException exc),
+    bool sleep() throws (1: ExpectedException exc),
```

Comment:
Always useful to have a slow version of the rpc when you're testing concurrency.